css/style.css

```diff
@@ -633,3 +631,356 @@ input[type="submit"]:active {
+    margin-top: 0;
+    width: 1156px;
+    height: 907px;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
+    gap: 1rem;
+  }
+
+  .cards-work-desktop {
+    width: 369px;
+    height: 435px;
+  }
+
+  .button-cards-work {
+    margin-top: 49px;
+  }
+
+  /* END Work Section */
+
+  /* BEGIN About Me Section */
+  .about-section-desktop {
+    width: 100%;
+    height: 1195px;
+    display: flex;
+    flex-direction: column;
+    background: #262626;
+    background-image: url(../images/illustration-about-me_2.png);
+    background-repeat: no-repeat;
+    background-position: 100% 10%;
+    margin-top: 8.12rem;
+  }
+
+  .about {
+    width: 566px;
+    height: 582px;
+    margin-left: 142px;
+  }
+
+  .vector-desktop {
+    margin-left: 24px;
+    margin-top: 209px;
+    width: 1156px;
+    border: 1px solid #fff;
+    background-color: #fff;
+  }
+
+  .about-skills-desktop {
+    width: 1158px;
+    height: 596px;
+    display: flex;
+    flex-direction: row;
+    gap: 162px;
+    align-items: flex-start;
+  }
+
+  .language-desktop {
+    width: 290px;
+    height: 282px;
+    margin-top: 138px;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 24px;
+  }
+
+  .frameworks-desktop {
+    width: 282px;
+    height: 336px;
+    margin-top: 138px;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 24px;
+  }
+
+  .skills-desktop {
+    width: 282px;
+    height: 336px;
+    margin-top: 138px;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 24px;
+  }
+
+  .language-desktop ul li {
+    text-align-last: justify;
+  }
+
+  .frameworks-desktop ul li {
+    text-align-last: justify;
+  }
+
+  .skills-desktop ul li {
+    text-align-last: justify;
+  }
+
+  /* END About Me Section */
+
+  /* BEGIN Contact Form Section */
+  .contact-form-mobile {
+    display: none;
+  }
+
+  .contact-form-container {
+    display: none;
+  }
+
+  .contact-form-desktop {
+    width: 327px;
+    height: 330px;
+    margin-left: 24px;
+    margin-bottom: 206px;
+    display: flex;
+    flex-direction: column;
+    justify-content: left;
+    font-family: Inter, sans-serif;
+    font-size: 15px;
+    font-weight: 400;
+    line-height: 24px;
+    text-align: left;
+    background-color: #fff;
+    color: #262626;
+  }
+
+  .contact-container {
+    width: 1163px;
+    height: 590px;
+    margin-top: 58px;
+    display: flex;
+    flex-direction: row;
+  }
+
+  .contact-form-desktop p {
+    width: 455px;
+    height: 256px;
+    margin-top: 0;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    display: flex;
+    align-items: center;
+  }
+
+  input {
+    width: 295px;
+    height: 18px;
+    border: 0;
+    margin-left: 16px;
+  }
+
+  input[type='email'] {
+    margin-top: 39px;
+  }
+
+  input[type='submit'] {
```

Comment:
-  [ ] I kindly suggest that the `Get in touch` button should be aligned to the left directly under the input forms

<img width="901" alt="image" src="https://user-images.githubusercontent.com/52934128/228741236-20e61062-9e81-4570-ac69-1f124201804d.png">